faq/sql-faq.md

```diff
@@ -298,7 +298,7 @@ In the preceding causes, only Cause 1 is related to tables. Cause 1 and Cause 2
 
 ### What are the causes of the "Information schema is out of date" error?
 
-Before TiDB v6.5.0, when executing a DML statement, if TiDB fails to load the latest schema within a DDL lease (45s by default), the `Information schema is out of date` error might occur. Possible causes are:
+When executing a DML statement, if TiDB fails to load the latest schema within a DDL lease (45s by default), the `Information schema is out of date` error might occur. Possible causes are:
```

Comment:
```suggestion
When executing a DML statement, if TiDB fails to load the latest schema within a DDL lease (45s by default), the `Information schema is out of date` error might occur. Possible causes are as follows:
```

Reply:
Introduce a list with a complete sentence, not a partial one that's completed by the list items. 
ref: https://developers.google.cn/style/lists#introductory-sentences-for-lists